src/Services/Utils/SQLUpgradeService.php

```diff
@@ -1025,8 +1025,9 @@ private function clickOptionsMigrate()
     private function CreateOccupationList()
     {
         $res = sqlStatement("SELECT DISTINCT occupation FROM patient_data WHERE occupation <> ''");
+        $records = [];
         while ($row = sqlFetchArray($res)) {
-            $records[] = $row['occupation'];
+            $records = $row['occupation'];
```

Comment:
isn't this supposed to create an array of occupations?
(i think these changes will just send back the last occupation as a string)